### ISSUE TITLE
"Try It" does not work for Anonymous authentication

### ISSUE THREAD

**Issue by AndrewBragdon** [mulesoft/api-console#549]
Here is what the configuration looks like:

![image](https://user-images.githubusercontent.com/5231939/34077160-164eb590-e2b1-11e7-899c-bc6a804894e5.png)

I get this error:

"Custom Security Schemes are not supported in Try It"

Here is a screenshot:

![image](https://user-images.githubusercontent.com/5231939/34077161-1f346d94-e2b1-11e7-8b3a-13eaf40c9fc3.png)

Using latest version of Anypoint Studio

**Comment by carowright** [comment]
Hi @AndrewBragdon,
Thanks for reporting. Could you please attach an example in order to reproduce this? Thank you.

**Comment by edson-19** [comment]
I got same error: "Custom Security Schemes are not supported in Try It "
I dont have custom security schemas
here is my api.raml:
#%RAML 1.0
baseUri: https://mocksvc.mulesoft.com/mocks/fe6c7949-ffcc-4761-95fc-a3c01bb3d95c
version: 1.0
mediaType: application/json


/contract:
  /{id}:
    get:
      responses: 
        200:
          body: 
            application/json:
              example: !include examples/getContractSummary.json

**Comment by carowright** [comment]
Hi @edson-19!
I couldn't get the error using latest console version (v3.0.30) and your spec (just without the last line). Are you using latest version? If so, please add steps to reproduce.

Regards,
Carolina

**Comment by edson-19** [comment]
hi Carolina, I noticed this error from Anypoint Studio ApiKit console,
but from browser http://127.0.0.1:8081/api/console/ is working fine
I have anypoint studio 6.5.0 and Mulesoft Server Runtime EE 3.9.1

how I can check the console version I have?
thanks for your support
Edson

**Comment by maquinas07** [comment]
I believe this has to do with incompatibilities with IE11. When using from the console in Studio 6.5.0 it throws the error

![](https://puu.sh/BsKLO/a5b12b933c.png)

And opening the console from IE11 shows the errors on the console when the function Object.assign is called

![](https://puu.sh/BsKQC/692915ed62.png)

Working fine on chrome

![](https://puu.sh/BsKRS/702a38ff7b.png)

**Comment by BujakiAttila** [comment]
I am having the same issue. I cannot make the API Console work properly. I tried several different security schema settings in the RAML file, but I keep on getting the same error message: "Custom Security Schemes are not supported in Try It"

I get this message, if there is no security schema defined and I also get this message, if I set it to basic for example.

The API-Kit console website can be loaded and the list of the available resources is also shown. The available methods (in my case GET) is also shown. There are no visible problems until I try to send a request. None of the sent "try-it" requests are successful and I keep on getting the same error message. (see above)

I am using Anypoint Studio Version 6.5.0 (Build Id: 201806071907)

I am using the following API-Kit Version (2018-04-23):

org.mule.tooling.server.3.9.1.ee_6.4.2.201804231909\mule\lib\mule\mule-module-apikit-3.9.1.jar 

There seems to be a related change in API Console 3.0.25:

https://docs.mulesoft.com/release-notes/api-console/api-console-release-notes#api-console-3-0-25

◦Fixed the issue causing API Console to throw "Custom Security Schemes are not supported in Try It”.

But this change should be actually included in my version. This version was released on January 13, 2018 and my version is from April.

There is only a single browser available on this computer: Internet Explorer 11 (11.0.9600.19.080)

IE 11 seems to be causing problems.

Is there eventually a newer version with better support for IE 11? In case not: are you planning a fix for this issue?

Thank you very much!
